README.md

```diff
@@ -1,6 +1,653 @@
-Active Directory Plugin
+Active Directory plugin for Jenkins
+===================================
 
-See wiki at: https://wiki.jenkins-ci.org/display/JENKINS/Active+Directory+plugin 
+[![Build Status](https://ci.jenkins.io/job/Plugins/job/active-directory-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/active-directory-plugin/job/master/)
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/active-directory.svg)](https://plugins.jenkins.io/active-directory/)
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/active-directory.svg?label=changelog)](https://github.com/jenkinsci/active-directory-plugin/releases/latest/)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/active-directory.svg?color=blue)](https://plugins.jenkins.io/active-directory/)
+
+| Plugin Information                                                                                            |
+|---------------------------------------------------------------------------------------------------------------|
+| View Active Directory [on the plugin site](https://plugins.jenkins.io/active-directory) for more information. |
```

Comment:
There is a No strong opinion, but there is a badge for it

Reply:
Removed the plugin information content ass requested by Mark and also resized the images.